BaseLib/ConfigTree.h

```diff
@@ -338,24 +338,6 @@ class ConfigTree final
      */
     //!\{
 
-    /*! Get parameter \c param from the configuration tree.
-     *
-     * \return the subtree representing the requested parameter
-     *
-     * \pre \c param must not have been read before from this ConfigTree.
-     */
-    ConfigTree
-    getConfParam(std::string const& param) const;
```

Comment:
Please keep these methods.
They are here on purpose: You might have something like '<tag attr="xyz">5.0</tag>` in an input file. I think it's more natural to parse that using `getConfParam()` rather than using `getConfSubtree()`.